modules/layers/src/icon-layer/icon-layer-fragment.glsl.js

```diff
@@ -38,7 +38,7 @@ void main(void) {
   // if colorMode == 0, use pixel color from the texture
   // if colorMode == 1 or rendering picking buffer, use texture as transparency mask
   vec3 color = mix(texColor.rgb, vColor.rgb, vColorMode);
-  float a = texColor.a * opacity * mix(1.0, vColor.a, vColorMode);
+  float a = texColor.a * opacity * vColor.a;
```

Comment:
Nit: I'd prefer calling this `alpha`

Reply:
Also, is this behavior clearly documented?

I can't see it here: http://deck.gl/#/documentation/deckgl-api-reference/layers/icon-layer?section=render-options

Also maybe a comment in the code here?